lib/io/reader.toit

```diff
@@ -761,27 +775,42 @@ abstract class CloseableReader extends Reader:
   abstract close_ -> none
 
 abstract mixin InMixin:
-  in_/In_? := null
+  in__/In_? := null
```

Comment:
Maybe use `_in_` instead for consistency with https://github.com/toitlang/toit/blob/master/lib/system/services.toit#L162?